docs/community/developer_guide.mdx

```diff
@@ -121,7 +136,7 @@ This section needs a lot of love and automation 🙂
         gpg --verify $file
     done
     ```
-1. Click the "Edit release" button on the release, and add the `.asc` files created in the previous step. Edit the release description for change logs and also add a note indicating your GPG key used to sign the artifacts. Example (replace with your fingerprint etc.):
+1. Click the "Edit release" button on the release, and add the `.asc` files created in the previous step. Edit the release description to include change logs and a note indicating releaser's GPG key used to sign the artifacts. Example (replace with releaser's fingerprint etc.):
     ```
```

Comment:
We have a https://oras.land/docs/how_to_guides/verifying_binaries document now, link it here somewhere?

Reply:
Added right after this action